build.rs

```diff
@@ -22,30 +22,42 @@ cfg_if::cfg_if! {
 }
 
 fn main() {
-    if let Err(err) = find_library() {
-        println!(
-            "cargo:warning={}: {}",
-            "Config for libftdi1 not found, falling back to default search path",
-            err,
-        );
-        println!("cargo:rustc-link-lib=dylib=ftdi1");
+    if cfg!(feature = "vendored") {
+        build_source();
+    } else {
+        if cfg!(feature = "libusb1-sys") {
+            match env::var("DEP_USB_1.0_STATIC") {
```

Comment:
Where is `DEP_USB_1.0_STATIC` defined? It doesn't look like it's used anywhere else, so I'm guessing an external crate defines it?

Reply:
Will be exported by `cargo` based on `libusb1-sys` build script output if/when a1ien/libusb1-sys#20 is accepted.